app/helpers/invitations_helper.rb

```diff
@@ -1,18 +1,20 @@
 module InvitationsHelper
-
   def creator_link(invitation)
-    if invitation.creator.is_a?(User)
+    case invitation.creator
+    when User
       link_to(invitation.creator.login, invitation.creator)
-    elsif invitation.creator.is_a?(Admin)
+    when Admin
```

Comment:
I might be wandering out of scope here, but while I was updating [AO3-6095](https://otwarchive.atlassian.net/browse/AO3-6095), I noticed it specifies using `Deleted admin` when the admin is deleted. We don't cover that case here, so is it possible this page would either 500 or incorrectly specify the queue if the admin who issued an invitation was now deleted?

Reply:
This refactor wouldn't change the behaviour mentioned in AO3-6095. I'd say it's out of scope for this PR, since this change here is just a rubocop thing